api/models/cartDao.js

```diff
@@ -85,17 +84,21 @@ const getCartByUserId = async(userId) => {
         INNER JOIN users u ON u.id= c.user_id
         WHERE user_id = ?`, [userId]
     )
-    return result[0] 
+    return result 
 }
 
-const deleteCartByCartId = async(cartId) => {
-    const deletedRows = (await dataSource.query(`
+const deleteCartByCartId = async(cartId, userId) => {
+    try{
+      const deletedRows = (await dataSource.query(`
             DELETE FROM carts
-            WHERE id = ?`, [cartId]
+            WHERE id = ? AND WHERE user_id = ?`, [cartId, userId]
```

Comment:
```suggestion
            WHERE id = ? AND user_id = ?`, [cartId, userId]
```

WHERE 구문을 AND 로 연결할 경우 두번 작성할 필요가 없습니다!